src/Interface.js

```diff
@@ -130,7 +130,11 @@ export function initialize (parent) {
  * @param {object} element
  */
 export function remove (parent, element) {
-	parent.removeChild(element)
+	try {
```

Comment:
surely swallowing the error here is not the correct solution (excluding cases of external DOM mutation)

Reply:
Two birds, one stone, external DOM mutation where one of the reasons.

Reply:
Would it make sense to log/throw anyways? Unless supporting (breaking) external DOM mutation is now a feature. My concern is that this leads to undefined behavior.

Reply:
>undefined behavior

Like?

Reply:
`removeChild` fails for any reason other than that the node is already removed. (Thus leaving the DOM in an 'inconsistent' state, where dyo (or other code) thinks it's gone but it's not). These types of bugs tend to lead to subtle and hard to debug issues.
All it needs is a `console.warn` I think.

Reply:
>fails for any reason other than that the node is already removed

Yes, it could also have been moved by a browser extension. But that is also within the range of things this is suppose to catch.

Reply:
Right, but you still want a `console.warn` in that situation

Reply:
Why? and how would that help if graceful support for external DOM mutation is the feature?

Reply:
Oh, sorry. I do _not_ think implicit support for external DOM mutation should be a feature (but acceptable that it only 'warns').
I'd rather see an explicit API, e.g. a `Static` node that guarantees that no DOM changes occur after initial render.

```coffee
Chart = ->
  useEffect ->
    # mount a chart to #chart'
    -> # unmount chart
  h 'div',
    h 'h1', 'my chart'
    h Static, # fragment
      h 'svg', id: 'chart'
```